prom2teams/app/exceptions.py

```diff
@@ -1,6 +1,10 @@
+from typing import Optional
```

Comment:
The class was not correctly initialized, my Neovim Python setup which is well configured shows this.

![image](https://github.com/idealista/prom2teams/assets/1327782/179297ac-d19f-4902-a2cf-73cd0cb000b0)

I fixed it and use typing which IMO should now always be used with recent Python versions.